cli/src/semgrep/commands/ci.py

```diff
@@ -385,6 +385,16 @@ def ci(
                 match for match in matches if not match.is_ignored or keep_ignored
             ]
         else:
+            if scan_handler:
```

Comment:
I think we should add a comment somewhere around here that mentions that blocking findings that have been triaged in UI count as non blocking findings

Reply:
Is that what is happening?